Cache.php

```diff
@@ -2,6 +2,8 @@
 
 namespace dcb9\redis;
 
+use Yii;
+use Exception;
```

Comment:
**question:** should we use `Throwable` instead?

Reply:
Yeah, you are right. In this case scenario we want to silence also runtime exceptions and other errors to silence it totally. Updating now the code.